GliaWidgets/Sources/Theme/Theme+Chat.swift

```diff
@@ -353,6 +353,8 @@ extension Theme {
             fileDownload: fileDownload
         )
 
+        let gvaStyle = gvaStyle
+
```

Comment:
could we pass `gvaStyle` directly to ChatStyle without this local variable?

Reply:
fixed [52e93dc](https://github.com/salemove/ios-sdk-widgets/pull/687/commits/52e93dc026b0a1e142e06d109309d8acec39728e)